test

```diff
@@ -57,11 +57,24 @@ fi
 split=(${TEST// / })
 TEST=${split[@]/#/${REPO_PATH}/}
 
+# PKGS_DELIM contains all the core etcd pkgs delimited by ',' which will be profiled for code coverage.
+# Integration and e2d tests will generate code coverage for those pkgs 
+export PKGS_DELIM=$(echo $TEST | sed 's/ /,/g')
+
 # determine whether target supports race detection
 if [ "$GOARCH" == "amd64" ]; then
 	RACE="--race"
 fi
 
+export COVEROUT="coverage"
```

Comment:
probably should let the user provide `COVERDIR=whatever/ PASSES=cov ./test` and not even bother with a default